question/bank/previewquestion/templates/preview_question.mustache

```diff
@@ -71,3 +78,5 @@
 {{#extrapreviewelements}}
```

Comment:
techinfo should be here and extra elements should come after that

Reply:
fixed